src/ensembl/src/content/app/entity-viewer/gene-view/GeneView.tsx

```diff
@@ -56,6 +63,9 @@ import { TicksAndScale } from 'src/content/app/entity-viewer/gene-view/component
 
 import { FullGene } from 'src/shared/types/thoas/gene';
 
+import { ReactComponent as ChevronDown } from 'static/img/shared/chevron-down.svg';
+import { ReactComponent as ChevronUp } from 'static/img/shared/chevron-up.svg';
```

Comment:
I thought we decided to use one svg and use css transform to rotate it?

Reply:
@jyothishnt  I am not aware of this but happy to make the change if this is whats been decided. Where is the code are we using the transform, might be easier to just copy from there :)

Reply:
As discussed in the code review, its better to use one svg as mentioned by @jyothishnt . I have updated the code to use chevron down and apply a transformation for it to be chevron up.

Reply:
Thanks Ridwan. Just saw this message. Hope you have found similar css transform on one of the chevrons we used.